include/ocpp/v201/functional_blocks/authorization.hpp

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#pragma once
+
+#include <ocpp/v201/messages/Authorize.hpp>
+#include <ocpp/v201/messages/ClearCache.hpp>
+#include <ocpp/v201/messages/GetLocalListVersion.hpp>
+#include <ocpp/v201/messages/SendLocalList.hpp>
+
+#include <ocpp/v201/database_handler.hpp>
+#include <ocpp/v201/message_dispatcher.hpp>
+#include <ocpp/v201/message_handler.hpp>
+
+namespace ocpp::v201 {
+class AuthorizationInterface : public MessageHandlerInterface {
+public:
+    virtual ~AuthorizationInterface() {
+    }
+
+    virtual void start() = 0;
```

Comment:
I would rename this with the same argument mentioned here https://github.com/EVerest/libocpp/pull/937/files#r1920169618